src/h_vialib/secure/url.py

```diff
@@ -40,9 +43,8 @@ def create(
         if not url:
             raise ValueError("A URL is required to create a token")
 
-        url = self._strip_token(url)
+        payload["h"] = self._hash_url_v1(url)
```

Comment:
I think we might want to add a comment explaining the `"h"` is used (instead of something like `"hashed_url"`) because we want to use short keys to keep the size of the JWT as short as possible. Probably means moving the `"h"` into a constant to provide a one-place where the comment can go

Reply:
I've updated this with an explanation